README.md

```diff
@@ -40,23 +36,13 @@ It's a collection of properties that contain the information necessary to create
 * `compose(...stampsOrDescriptors) => stamp` **Creates stamps.** Take any number of stamps or descriptors.
 Return a new stamp that encapsulates combined behavior. If nothing is passed in, it returns an empty stamp.
 
------
```

Comment:
Ah, I knew there was something I meant to move, but never found a place for... Good catch!